web/src/app/services/user.service.ts

```diff
@@ -42,7 +44,7 @@ export class UserService {
 
   getUser(): Observable<User> {
     return this.authService.user.pipe(
-      map(response => this.formatResponse(response))
+      map(response => this.formatResponse(response)),
```

Comment:
Ending comma is inconsistent throughout the project, for example further down on line `64`. I think the recommendation is to have an ending comma

Reply:
Okay @eddiejaoude . Will change it and make it consistent 👍

Reply:
👍